src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.html

```diff
@@ -1,31 +1,26 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-one-third">
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
-        <app-tech-record-title
-          [vehicleTechRecord]="vehicleTechRecord"
-          [recordActions]="getActions((currentTechRecord$ | async) || undefined)"
-        ></app-tech-record-title>
+      <div class="govuk-grid-column-full" *ngIf="currentTechRecord$ | async as currentTechRecord">
+        <app-tech-record-title [vehicleTechRecord]="vehicleTechRecord" [recordActions]="getActions(currentTechRecord)"></app-tech-record-title>
 
         <app-edit-tech-record-button
           [vehicleTechRecord]="vehicleTechRecord"
-          [viewableTechRecord]="(currentTechRecord$ | async) || undefined"
+          [viewableTechRecord]="currentTechRecord"
           [(isEditing)]="isEditing"
           [isDirty]="isDirty"
           (submitChange)="handleSubmit()"
           *appRoleRequired="roles.TechRecordAmend"
         ></app-edit-tech-record-button>
 
-        <app-tech-record-history
-          [currentRecord]="(currentTechRecord$ | async) || undefined"
-          [vehicleTechRecord]="vehicleTechRecord"
-        ></app-tech-record-history>
+        <app-tech-record-history [currentRecord]="currentTechRecord" [vehicleTechRecord]="vehicleTechRecord"></app-tech-record-history>
 
-        <app-test-record-summary *appRoleRequired="roles.TestResultView" [testRecords]="(records$ | async) || []"> </app-test-record-summary>
+        <app-test-record-summary *appRoleRequired="roles.TestResultView" [testRecords]="(records$ | async) || []"></app-test-record-summary>
 
         <ng-container *ngIf="!isArchived && !isEditing">
-          <app-test-record-create-button *appRoleRequired="[roles.TestResultCreateContingency, roles.TestResultCreateDeskAssesment]">
-          </app-test-record-create-button>
+          <div class="govuk-button-group" *appRoleRequired="[roles.TestResultCreateContingency, roles.TestResultCreateDeskAssesment]">
+            <a id="create-test" class="govuk-button" (click)="createTest(currentTechRecord?.recordCompleteness)">Create a test for this vehicle</a>
```

Comment:
This should be a button, or it should have an `href` or `routerLink` attribute.

Reply:
The hover and cursor effect is already handled by the gov styling, so it is not necessary.